Doc/howto/clinic.rst

```diff
@@ -162,28 +162,20 @@ For the sake of clarity, here's the terminology we'll use with Argument Clinic:
 Tutorial
 ========
 
-The best way to get a sense of how Argument Clinic works is to
-convert a function to work with it.  Here, then, are the bare
-minimum steps you'd need to follow to convert a function to
-work with Argument Clinic.  Note that for code you plan to
-check in to CPython, you really should take the conversion farther,
-using some of the :ref:`advanced concepts <clinic-howtos>`
-you'll see later on in the document,
-like :ref:`clinic-howto-return-converters`
-and :ref:`clinic-howto-self-converter`.
-But we'll keep it simple for this walkthrough so you can learn.
-
-First, make sure you're working with a freshly updated checkout
-of the CPython trunk.
-
-Next, find a Python builtin that calls either :c:func:`PyArg_ParseTuple`
+In this tutorial, you will convert a C function from having custom
+:ref:`argument parsing code <arg-parsing>`
+to using Argument Clinic to generate that boilerplate code.
+It assumes a fundamental understanding of the C APIs and format strings used
+for :ref:`arg-parsing`.
+
+First, find a Python builtin that calls either :c:func:`PyArg_ParseTuple`
 or :c:func:`PyArg_ParseTupleAndKeywords`, and hasn't been converted
 to work with Argument Clinic yet.
-For this tutorial, we'll be using
-:py:meth:`_pickle.Pickler.dump <pickle.Pickler.dump>`.
+We'll be using :py:meth:`_pickle.Pickler.dump <pickle.Pickler.dump>`
+as our example C function in this tutorial.
 
-If the call to the :c:func:`!PyArg_Parse*` function uses any of the
-following format units...:
+Next, take a look at the format string of the :c:func:`!PyArg_Parse*` call.
+If it contains any of the following format units...:
```

Comment:
We should consider removing this from the tutorial, in my opinion. Alternatives and information are tutorial anti-patterns.

Reply:
You could state what format units are relevant for the tutorial, and link to the advanced converters section for the others.  IOW state what works, rather than what doesn't work.